### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by gingerbeardman** [dosdude1/discord-lite#4]
I have 2FA activated on my Discord account.

Crash log
https://pastebin.com/raw/n2RaXEqx

Screenshots

<img width="402" alt="Screen shot 2021-12-11 at 13 20 09" src="https://user-images.githubusercontent.com/49612/145678236-c9452445-6f0e-4b3e-a616-8efbd6707b1c.png">

<img width="1050" alt="Screen shot 2021-12-11 at 13 19 36" src="https://user-images.githubusercontent.com/49612/145678233-7908cfdb-097f-45de-a4ee-784b922c4d79.png">

**Comment by dosdude1** [comment]
Yes, that's most likely due to 2FA, as I am not properly handling the response for 2FA in the client. I will most likely have this implemented in the next release.

**Comment by gingerbeardman** [comment]
I temporarily disabled 2FA, and login just hangs for me: #5
